<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TestResultsAnalysisWindow</class>
 <widget class="QWidget" name="TestResultsAnalysisWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1350</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>测评结果分析模块</string>
  </property>
  <property name="styleSheet">
   <string notr="true">
    background-color: #f5f7fa;
    font-family: &quot;Microsoft YaHei&quot;, &quot;Segoe UI&quot;, sans-serif;
   </string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>12</number>
   </property>
   <property name="leftMargin">
    <number>12</number>
   </property>
   <property name="topMargin">
    <number>12</number>
   </property>
   <property name="rightMargin">
    <number>12</number>
   </property>
   <property name="bottomMargin">
    <number>12</number>
   </property>
   <item>
    <widget class="QWidget" name="filterWidget" native="true">
     <property name="styleSheet">
      <string notr="true">
       background-color: white;
       border-radius: 6px;
       padding: 8px;
       box-shadow: 0 1px 4px rgba(0,0,0,0.05);
      </string>
     </property>
     <layout class="QHBoxLayout" name="filterLayout">
      <property name="spacing">
       <number>10</number>
      </property>
      <property name="leftMargin">
       <number>4</number>
      </property>
      <property name="topMargin">
       <number>4</number>
      </property>
      <property name="rightMargin">
       <number>4</number>
      </property>
      <property name="bottomMargin">
       <number>4</number>
      </property>
      <item>
       <widget class="QLabel" name="labelTask">
        <property name="styleSheet">
         <string notr="true">color: #000000; font-size: 13px;</string>
        </property>
        <property name="text">
         <string>测试任务:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="comboTask">
        <property name="styleSheet">
         <string notr="true">
          QComboBox {
              padding: 6px 12px;
              border: 1px solid #d0d0d0;
              border-radius: 4px;
              min-width: 150px;
              font-size: 13px;
              color: #000000;
          }
          QComboBox::drop-down {
              width: 20px;
              border: none;
              color: #000000;
          }
          QComboBox QAbstractItemView {
              color: #000000;  /* 下拉项文字颜色 */
              background-color: white;  /* 下拉项背景色 */
              selection-color: white;  /* 选中项文字颜色 */
              selection-background-color: #3a6ea5;  /* 选中项背景色 */
              outline: 0;
              border: 1px solid #d0d0d0;
              font-size: 13px;
          }
         </string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="labelUnit">
        <property name="styleSheet">
         <string notr="true">color: #000000; font-size: 13px;</string>
        </property>
        <property name="text">
         <string>参试单位:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="comboUnit">
        <property name="styleSheet">
         <string notr="true">
         QComboBox {
             padding: 6px 12px;
             border: 1px solid #d0d0d0;
             border-radius: 4px;
             min-width: 140px;
             font-size: 13px;
             color: #000000;
         }
         QComboBox::drop-down {
             width: 20px;
             border: none;
             color: #000000;
         }
         QComboBox QAbstractItemView {
             color: #000000;  /* 下拉项文字颜色 */
             background-color: white;  /* 下拉项背景色 */
             selection-color: white;  /* 选中项文字颜色 */
             selection-background-color: #3a6ea5;  /* 选中项背景色 */
             outline: 0;
             border: 1px solid #d0d0d0;
             font-size: 13px;
         }
         </string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="labelTime">
        <property name="styleSheet">
         <string notr="true">color: #333333; font-size: 13px;</string>
        </property>
        <property name="text">
         <string>时间范围:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDateEdit" name="dateStart">
        <property name="styleSheet">
         <string notr="true">color: rgb(0, 0, 0);
          QDateEdit {
              padding: 6px 12px;
              border: 1px solid #d0d0d0;
              border-radius: 4px;
              min-width: 120px;
              font-size: 13px;
          }
         </string>
        </property>
        <property name="calendarPopup">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="labelTo">
        <property name="styleSheet">
         <string notr="true">color: #333333; font-size: 13px;</string>
        </property>
        <property name="text">
         <string>至</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDateEdit" name="dateEnd">
        <property name="styleSheet">
         <string notr="true">color: rgb(0, 0, 0);
          QDateEdit {
              padding: 6px 12px;
              border: 1px solid #d0d0d0;
              border-radius: 4px;
              min-width: 120px;
              font-size: 13px;
          }
         </string>
        </property>
        <property name="calendarPopup">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Orientation::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="btnQuery">
        <property name="styleSheet">
         <string notr="true">
          QPushButton {
              background-color: #3a6ea5;
              color: white;
              border: none;
              padding: 8px 16px;
              border-radius: 4px;
              font-size: 13px;
              min-width: 80px;
          }
          QPushButton:hover {
              background-color: #4a90e2;
          }
          QPushButton:pressed {
              background-color: #2a5685;
          }
         </string>
        </property>
        <property name="text">
         <string>查询</string>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>:/icons/search.png</normaloff>:/icons/search.png</iconset>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnExport">
        <property name="styleSheet">
         <string notr="true">
          QPushButton {
              background-color: #27ae60;
              color: white;
              border: none;
              padding: 8px 16px;
              border-radius: 4px;
              font-size: 13px;
              min-width: 80px;
          }
          QPushButton:hover {
              background-color: #2ecc71;
          }
          QPushButton:pressed {
              background-color: #219955;
          }
         </string>
        </property>
        <property name="text">
         <string>导出</string>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>:/icons/export.png</normaloff>:/icons/export.png</iconset>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="styleSheet">
      <string notr="true">
       QTabWidget::pane {
           border: 1px solid #e0e0e0;
           border-radius: 6px;
           background: white;
       }
       QTabBar::tab {
           background: #f0f0f0;
           border: 1px solid #e0e0e0;
           border-bottom: none;
           border-top-left-radius: 4px;
           border-top-right-radius: 4px;
           padding: 8px 16px;
           font-size: 13px;
           color: #333333;
       }
       QTabBar::tab:selected {
           background: white;
           border-bottom: 2px solid #3a6ea5;
           color: #3a6ea5;
           font-weight: 500;
       }
       QTabBar::tab:hover {
           background: #e9ecef;
       }
      </string>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tabSummary">
      <attribute name="title">
       <string>总体分析</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <widget class="QWidget" name="widgetPassRate" native="true">
         <property name="styleSheet">
          <string notr="true">
           background-color: white;
           border-radius: 6px;
           box-shadow: 0 1px 4px rgba(0,0,0,0.05);
          </string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <property name="spacing">
           <number>6</number>
          </property>
          <property name="leftMargin">
           <number>8</number>
          </property>
          <property name="topMargin">
           <number>8</number>
          </property>
          <property name="rightMargin">
           <number>8</number>
          </property>
          <property name="bottomMargin">
           <number>8</number>
          </property>
          <item>
           <widget class="QLabel" name="labelPassRate">
            <property name="styleSheet">
             <string notr="true">color: #333333; font-size: 14px; font-weight: 500;</string>
            </property>
            <property name="text">
             <string>测试通过率统计</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QChartView" name="chartViewPassRate" native="true">
            <property name="styleSheet">
             <string notr="true">background: transparent;</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QWidget" name="widgetItemDistribution" native="true">
         <property name="styleSheet">
          <string notr="true">
           background-color: white;
           border-radius: 6px;
           box-shadow: 0 1px 4px rgba(0,0,0,0.05);
          </string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <property name="spacing">
           <number>6</number>
          </property>
          <property name="leftMargin">
           <number>8</number>
          </property>
          <property name="topMargin">
           <number>8</number>
          </property>
          <property name="rightMargin">
           <number>8</number>
          </property>
          <property name="bottomMargin">
           <number>8</number>
          </property>
          <item>
           <widget class="QLabel" name="labelItemDistribution">
            <property name="styleSheet">
             <string notr="true">color: #333333; font-size: 14px; font-weight: 500;</string>
            </property>
            <property name="text">
             <string>测试项分布</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QChartView" name="chartViewItemDistribution" native="true">
            <property name="styleSheet">
             <string notr="true">background: transparent;</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QWidget" name="widgetUnitPerformance" native="true">
         <property name="styleSheet">
          <string notr="true">
           background-color: white;
           border-radius: 6px;
           box-shadow: 0 1px 4px rgba(0,0,0,0.05);
          </string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <property name="spacing">
           <number>6</number>
          </property>
          <property name="leftMargin">
           <number>8</number>
          </property>
          <property name="topMargin">
           <number>8</number>
          </property>
          <property name="rightMargin">
           <number>8</number>
          </property>
          <property name="bottomMargin">
           <number>8</number>
          </property>
          <item>
           <widget class="QLabel" name="labelUnitPerformance">
            <property name="styleSheet">
             <string notr="true">color: #333333; font-size: 14px; font-weight: 500;</string>
            </property>
            <property name="text">
             <string>参试单位表现</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QChartView" name="chartViewUnitPerformance" native="true">
            <property name="styleSheet">
             <string notr="true">background: transparent;</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QWidget" name="widgetTimeTrend" native="true">
         <property name="styleSheet">
          <string notr="true">
           background-color: white;
           border-radius: 6px;
           box-shadow: 0 1px 4px rgba(0,0,0,0.05);
          </string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_5">
          <property name="spacing">
           <number>6</number>
          </property>
          <property name="leftMargin">
           <number>8</number>
          </property>
          <property name="topMargin">
           <number>8</number>
          </property>
          <property name="rightMargin">
           <number>8</number>
          </property>
          <property name="bottomMargin">
           <number>8</number>
          </property>
          <item>
           <widget class="QLabel" name="labelTimeTrend">
            <property name="styleSheet">
             <string notr="true">color: #333333; font-size: 14px; font-weight: 500;</string>
            </property>
            <property name="text">
             <string>时间趋势分析</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QChartView" name="chartViewTimeTrend" native="true">
            <property name="styleSheet">
             <string notr="true">background: transparent;</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabDetails">
      <attribute name="title">
       <string>详细数据</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_6">
       <property name="spacing">
        <number>8</number>
       </property>
       <property name="leftMargin">
        <number>8</number>
       </property>
       <property name="topMargin">
        <number>8</number>
       </property>
       <property name="rightMargin">
        <number>8</number>
       </property>
       <property name="bottomMargin">
        <number>8</number>
       </property>
       <item>
        <widget class="QTableWidget" name="tableResults">
         <property name="styleSheet">
          <string notr="true">
           QTableWidget {
               font-size: 13px;
               background-color: white;
               alternate-background-color: #f8f9fa;
               border: 1px solid #e0e0e0;
               border-radius: 6px;
               gridline-color: #e0e0e0;
               outline: 0;
           }
           QTableWidget::item {
               padding: 8px;
               border-bottom: 1px solid #f0f0f0;
               color: #333333;
           }
           QTableWidget::item:hover {
               background-color: #f0f7ff;
           }
           QHeaderView::section {
               background-color: #3a6ea5;
               color: white;
               padding: 8px;
               border: none;
               font-weight: 500;
               font-size: 13px;
           }
           QHeaderView {
               border-top-left-radius: 6px;
               border-top-right-radius: 6px;
           }
          </string>
         </property>
         <property name="sizeAdjustPolicy">
          <enum>QAbstractScrollArea::SizeAdjustPolicy::AdjustToContents</enum>
         </property>
         <property name="alternatingRowColors">
          <bool>true</bool>
         </property>
         <property name="selectionMode">
          <enum>QAbstractItemView::SelectionMode::NoSelection</enum>
         </property>
         <property name="selectionBehavior">
          <enum>QAbstractItemView::SelectionBehavior::SelectRows</enum>
         </property>
         <property name="showGrid">
          <bool>false</bool>
         </property>
         <property name="sortingEnabled">
          <bool>true</bool>
         </property>
         <property name="rowCount">
          <number>0</number>
         </property>
         <property name="columnCount">
          <number>7</number>
         </property>
         <column>
          <property name="text">
           <string>测试任务</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>参试单位</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>测试项</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>标准要求</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>测试结果</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>结果描述</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>测试时间</string>
          </property>
         </column>
        </widget>
       </item>
       <item>
        <widget class="QWidget" name="paginationWidget" native="true">
         <property name="styleSheet">
          <string notr="true">
           background-color: white;
           border-radius: 6px;
           padding: 8px;
           box-shadow: 0 1px 4px rgba(0,0,0,0.05);
          </string>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <property name="spacing">
           <number>8</number>
          </property>
          <property name="leftMargin">
           <number>8</number>
          </property>
          <property name="topMargin">
           <number>4</number>
          </property>
          <property name="rightMargin">
           <number>8</number>
          </property>
          <property name="bottomMargin">
           <number>4</number>
          </property>
          <item>
           <widget class="QLabel" name="labelPageInfo">
            <property name="styleSheet">
             <string notr="true">
             color: #333333;
             font-size: 13px;
             font-weight: 500;
             </string>
            </property>
            <property name="text">
             <string>第 1 页 / 共 1 页 (10 条)</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_2">
            <property name="orientation">
             <enum>Qt::Orientation::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="btnFirst">
            <property name="styleSheet">
             <string notr="true">
              QPushButton {
                  min-width: 60px;
                  padding: 6px 8px;
                  background-color: #f8f9fa;
                  border: 1px solid #d0d0d0;
                  border-radius: 4px;
                  font-size: 12px;
              }
              QPushButton:hover {
                  background-color: #e9ecef;
              }
              QPushButton:pressed {
                  background-color: #dee2e6;
              }
             </string>
            </property>
            <property name="text">
             <string>首页</string>
            </property>
            <property name="icon">
             <iconset>
              <normaloff>:/icons/first.png</normaloff>:/icons/first.png</iconset>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btnPrev">
            <property name="styleSheet">
             <string notr="true">
              QPushButton {
                  min-width: 60px;
                  padding: 6px 8px;
                  background-color: #f8f9fa;
                  border: 1px solid #d0d0d0;
                  border-radius: 4px;
                  font-size: 12px;
              }
              QPushButton:hover {
                  background-color: #e9ecef;
              }
              QPushButton:pressed {
                  background-color: #dee2e6;
              }
             </string>
            </property>
            <property name="text">
             <string>上一页</string>
            </property>
            <property name="icon">
             <iconset>
              <normaloff>:/icons/prev.png</normaloff>:/icons/prev.png</iconset>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="comboPage">
            <property name="styleSheet">
             <string notr="true">
              QComboBox {
                  padding: 4px 8px;
                  border: 1px solid #d0d0d0;
                  border-radius: 4px;
                  min-width: 60px;
                  font-size: 12px;
              }
              QComboBox::drop-down {
                  width: 20px;
                  border: none;
              }
             </string>
            </property>
            <property name="currentText">
             <string/>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btnNext">
            <property name="styleSheet">
             <string notr="true">
              QPushButton {
                  min-width: 60px;
                  padding: 6px 8px;
                  background-color: #f8f9fa;
                  border: 1px solid #d0d0d0;
                  border-radius: 4px;
                  font-size: 12px;
              }
              QPushButton:hover {
                  background-color: #e9ecef;
              }
              QPushButton:pressed {
                  background-color: #dee2e6;
              }
             </string>
            </property>
            <property name="text">
             <string>下一页</string>
            </property>
            <property name="icon">
             <iconset>
              <normaloff>:/icons/next.png</normaloff>:/icons/next.png</iconset>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btnLast">
            <property name="styleSheet">
             <string notr="true">
              QPushButton {
                  min-width: 60px;
                  padding: 6px 8px;
                  background-color: #f8f9fa;
                  border: 1px solid #d0d0d0;
                  border-radius: 4px;
                  font-size: 12px;
              }
              QPushButton:hover {
                  background-color: #e9ecef;
              }
              QPushButton:pressed {
                  background-color: #dee2e6;
              }
             </string>
            </property>
            <property name="text">
             <string>末页</string>
            </property>
            <property name="icon">
             <iconset>
              <normaloff>:/icons/last.png</normaloff>:/icons/last.png</iconset>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="comboPageSize">
            <property name="styleSheet">
             <string notr="true">
              QComboBox {
                  padding: 4px 8px;
                  border: 1px solid #d0d0d0;
                  border-radius: 4px;
                  min-width: 80px;
                  font-size: 12px;
              }
              QComboBox::drop-down {
                  width: 20px;
                  border: none;
              }
             </string>
            </property>
            <item>
             <property name="text">
              <string>10 条/页</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>20 条/页</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>50 条/页</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>100 条/页</string>
             </property>
            </item>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QChartView</class>
   <extends>QWidget</extends>
   <header>qchartview.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
